qadence_protocols/types.py

```diff
@@ -14,6 +18,14 @@ def list(cls) -> list[str]:
         return list(map(lambda c: c.value, cls))  # type: ignore
 
 
+@dataclass
+class MeasurementData:
+    measurements: Tensor | list[Counter] | None = None
+    """Measurements data."""
+    unitaries: Tensor | None = None
```

Comment:
I'd default them to empty tensors or empty lists. it'll avoid the handling of the `None` case I think.

Reply:
Just wondering if the measurements field shouldn't be renamed `samples` and typed as a `list[Counter]` only.

Reply:
Hm I can do but tensors are better suited for the shadows IMO.

Reply:
Removed None